django/db/models/query.py

```diff
@@ -728,7 +728,7 @@ def _check_bulk_create_options(
                     "bulk_create() can only be used with concrete fields in "
                     "update_fields."
                 )
-            if any(f.primary_key for f in update_fields):
```

Comment:
This relates to ticket-36068 (#19011) and solves the issues without branching off `is_composite_pk`.